content/news/029/index.md

```diff
@@ -75,6 +75,18 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library Updates
 
+### [Shard]
+
+[Shard] ([GitHub]) by @HindrikStegenga]
+is an Archetype-based Entity Component System implemented in Rust.
+
+Version 0.2 is a complete rewrite of the ECS, with the main new feature that it supports no_std environments.
+
+_Discussions: [/r/rust](https://www.reddit.com/r/rust/comments/r6mn0g/announcing_shard_020_the_no_std_entity_component/)_
```

Comment:
Links moved to the bottom please :)

Reply:
FWIW this is how the link is formatted in `newsletter-template` too:

![image](https://user-images.githubusercontent.com/784533/148834334-ec937eea-5d94-4558-85bd-2ee811e2d95e.png)

Whether that's by design or by mistake, I'm not sure!

Reply:
Yeah, that was intentional. I'm not 100% sure, but in case of discussion links they seem to format nicely at their own lines and twi/reddit links are usually short enough